wallet/src/Pos/Wallet/Web/Methods/Backup.hs

```diff
@@ -5,84 +5,29 @@
 module Pos.Wallet.Web.Methods.Backup
        ( importWalletJSON
        , exportWalletJSON
-       , -- | just for testing
-         restoreWalletFromBackup
        ) where
 
 import           Universum
 
-import           Control.Lens (each)
 import qualified Data.Aeson as A
 import qualified Data.ByteString.Lazy as BSL
-import qualified Data.HashMap.Strict as HM
 import           Formatting (sformat, stext, (%))
 
-import           Pos.Client.KeyStorage (addSecretKey)
-import           Pos.Crypto (emptyPassphrase, firstHardened)
-import           Pos.Wallet.Web.Account (GenSeed (..), genUniqueAccountId)
-import           Pos.Wallet.Web.Backup (AccountMetaBackup (..), TotalBackup (..), WalletBackup (..),
-                                        WalletMetaBackup (..), getWalletBackup)
-import           Pos.Wallet.Web.ClientTypes (CAccountInit (..), CAccountMeta (..), CFilePath (..),
-                                             CId, CWallet, Wal, encToCId)
+import           Pos.Util (HasLens (..))
+import           Pos.Wallet.Web.Backup (TotalBackup (..), getWalletBackup)
+import           Pos.Wallet.Web.ClientTypes (CFilePath (..), CId, CWallet, Wal)
 import           Pos.Wallet.Web.Error (WalletError (..))
 import qualified Pos.Wallet.Web.Methods.Logic as L
-import           Pos.Wallet.Web.State (AddressLookupMode (Ever), askWalletDB, askWalletSnapshot,
-                                       createAccount, getAccountWAddresses, getWalletMeta,
-                                       getWalletSnapshot)
-import           Pos.Wallet.Web.Tracking (syncWalletOnImport)
-import           Pos.Wallet.Web.Util (getWalletAccountIds)
+import           Pos.Wallet.Web.Methods.Restore (restoreWalletFromBackup)
+import           Pos.Wallet.Web.State.State (askWalletSnapshot)
+import           Pos.Wallet.Web.Tracking.Types (SyncQueue)
 import           Servant.API.ContentTypes (NoContent (..))
+import           UnliftIO (MonadUnliftIO)
 
-
```

Comment:
This code has been moved from `Backup` to `Restore` -- the intention was to have a more cohesive module with "all-the-thing-restoration".